### DISCUSSION TITLE
关于目录标题上的红框

### DISCUSSION THREAD

**Discussion by alex-123123** [sjtug/SJTUThesis#887] [提问] [answered]
下载了新的sjtu模板，使用了vscode编译以后发现目录标题和页码上都有红框，以前的模板好像没有这个情况，请问一下该如何去掉
![image](https://user-images.githubusercontent.com/123817955/215270769-54c7184f-8648-4f56-bb8a-8b788fcd17e2.png)

**Comment by LogCreative** [comment, marked as answer]
红框到最后不会被打印出来，如想去除，请将该行

https://github.com/sjtug/SJTUThesis/blob/838c0e99dff5105c92ac08d5b7ed3f10c0318e1c/setup.tex#L194

更改为

```latex
\usepackage[hidelinks]{hyperref}
```

**Reply by alex-123123** [reply]
多谢多谢
